packages/Toast/stories/examples/UncontrolledToastExample.js

```diff
@@ -1,15 +1,30 @@
 import React from "react";
 
+import CodeViewer from "storybook/components/CodeViewer";
 import L10n from "@paprika/l10n";
 import Toast from "../../src";
 
 export default function UncontrolledToast() {
+  const [accumulator, setAccumulator] = React.useState(0);
+
+  function handleItem() {
+    setAccumulator(prev => prev + 1);
+  }
+
   return (
     <L10n locale="en">
       <Toast hasCloseButton>Uncontrolled toast component</Toast>
       <Toast canAutoClose autoCloseDelay={5000} hasCloseButton>
```

Comment:
A little out of scope, but while we're here, I think it makes sense to clean up this story just a little.  There are some default props added here that are not actually needed, which is a distraction from the use case being illustrated.